Source/Response.swift

```diff
@@ -77,6 +88,7 @@ extension Response: CustomDebugStringConvertible {
         output.append(response != nil ? "[Response]: \(response!)" : "[Response]: nil")
         output.append("[Data]: \(data?.length ?? 0) bytes")
         output.append("[Result]: \(result.debugDescription)")
+        output.append("[Timeline]: \(timeline.debugDescription)")
```

Comment:
This is possible if the value stored in the result contains newlines. There's nothing to fix here.